### ISSUE TITLE
Stuck at "Update UTMP about System Runlevel Changes"

### ISSUE THREAD

**Issue by 21Development** [hoobs-org/hoobsd#61]
HOOBS v4.0.23 (link from Beta email) flashed to a 32GB card via balenaEtcher inserted into Raspberry Pi B+ connected via RJ45

Pi boots, resizes partition then restarts
Displays login prompt with 127.0.0.1 as one of the options to connect, a few seconds later it refreshes with the 192.x IP address its assigned.

After a minute or so get Stopped/Started messages as per screenshot, then hangs at "Update UTMP about System Runlevel Changes.."

Browsing to the 192 IP address gives a refused to connect error in Chrome. I can ping the IP.

![HOOBS 4 0 23](https://user-images.githubusercontent.com/39352639/117844134-590feb80-b277-11eb-94cb-2a284ec24a56.jpg)

Left for about 45 mins, then rebooted. This time stops at the "hoobs login: " prompt - I can SSH into it but still cannot access via a web browser.

I have tried reflashing the card and got the exact same results.

**Comment by 21Development** [comment]
I see there are later versions available, how can I update / get the latest version? (4.0.23 was downloaded from the link in the beta email I received).

**Comment by 21Development** [comment]
The Pi is connected via cable (not wifi) and I can ping (consistent 20ms pings) and SSH to it when it's booted, so it has access to the network/Internet.

I've reflashed the card three times now, it verify's okay each time by balenaEtcher after writing the image and does the same thing each time on 1st boot and subsequent boots.

For reference, I've just inserted my original mSD card (v3.x) and during boot that also fails the update "Timezone Update Service" but that HOOBS been working okay for the last year or so.

I've flashed to a different 32GB microSD card and get the same results.

**Comment by mkellsy** [comment]
SSH into the device and run this
```
sudo journalctl -o cat -n 500 -f -u hoobsd.service
```

**Comment by 21Development** [comment]
The journalctl command gives this multiple times

> hoobsd.service: Main process exited, code=killed, status=4/ILL
hoobsd.service: Failed with result 'signal'.
hoobsd.service: Service RestartSec=3s expired, scheduling restart.
hoobsd.service: Scheduled restart job, restart counter is at 586.

> hoobs@hoobs:~ $ uname -a
Linux hoobs 5.10.17+ #1414 Fri Apr 30 13:16:27 BST 2021 armv6l GNU/Linux

This is a Raspberry Pi B+ as stated in original message, is this the reason? Is there a version for this Pi?

> hoobs@hoobs:~` $ node -v
Illegal instruction

**Comment by 21Development** [comment]
The current version of HOOBS supports both architectures, is the official answer that HOOBS is no longer supporting the millions of "Raspberry Pi 1B, 2B and Zero W" devices with version 4?? - Is there some functionality in v4 that requires more powerful processing? Requires Wifi? or are you just looking to reduce your development costs?

**Comment by mkellsy** [comment]
Node stopped officially supporting armv6l.
You can still build Node for armv6l or find an unsupported binary and manually install.

Unfortunately we needed a managed solution for Node. Since armv6l is not supported, we had to drop it as well.

**Comment by 21Development** [comment]
Node lowered it's support level for Armv6 two years ago (March 2019) but is still updated here https://unofficial-builds.nodejs.org/download/release/ and you have continued to release v3.x versions of HOOBS since then.

If your company drops support for hardware without notice (theres no mention of it on the beta email or your website), sounds like I should drop HOOBS and use the original https://github.com/homebridge/homebridge-raspbian-image (which fixes some of the bugs in v3.3.3) or use the opportunity to see what else is out there now.

**Comment by mkellsy** [comment]
You are welcome to install Node manually using the unofficial builds. However, we moved to `apt-get install nodejs` using the Nodesource repos, which doesn't support armv6l (armel). This makes upgrading easy for 99+% of our user base. Upgrading Node has been a pain point for our mainstream users and we needed to solve this problem.

HOOBS 4 includes Node upgrades in the UI now thanks to APT and the Nodesource repos. We have had massive positive feedback on this. I do understand that writing software is a game of balance, you win some and loose some.

We didn't make this decision lightly. I even thought of adding Node to our apt repo, which we may still do in the future. As of this beta release, we are focused on our code and not maintaining a Node APT repository that contains ARM v6.
